apps/prairielearn/src/pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.ts

```diff
@@ -150,7 +151,8 @@ export const InstructorSharing = ({
                   <td data-testid="sharing-name">
                     ${sharingName !== null
                       ? sharingName
-                      : html`
+                      : isCourseOwner
+                      ? html`
```

Comment:
This nested ternary is difficult to parse. Consider splitting this into two separate conditionals.

Reply:
sure, done